helm/config-controller/templates/service.yaml

```diff
@@ -5,6 +5,7 @@ metadata:
   namespace: {{ include "resource.default.namespace"  . }}
   labels:
     {{- include "labels.common" . | nindent 4 }}
+    giantswarm.io/monitoring: "true"
```

Comment:
You also need to configure `giantswarm.io/monitoring-path" and "giantswarm.io/monitoring-port" for the new setup to pick this up

Reply:
I thought there are defaults?

Reply:
Sadly no, we have a stuck PR about this: https://github.com/giantswarm/prometheus-meta-operator/pull/245

Reply:
TBH I even prefer to be explicit. I'll fix it.